wpilibNewCommands/src/main/native/cpp/frc2/command/CommandScheduler.cpp

```diff
@@ -53,10 +53,10 @@ class CommandScheduler::Impl {
 
   // Flag and queues for avoiding concurrent modification if commands are
   // scheduled/canceled during run
-
```

Comment:
?

Reply:
Well, there's no reason to have an extra line there. I can switch it back if preferred.